nanome/api/shapes/label.py

```diff
@@ -0,0 +1,35 @@
+from nanome._internal._shapes import _Label
+from nanome.util.enums import ShapeType
+from . import Shape
+
+
+class Label(_Label, Shape):
+    def __init__(self):
+        Shape.__init__(self, ShapeType.Label)
```

Comment:
doesn't `_Label.__init__` already call `_Shape.__init__`? same for the other shapes' inits

Reply:
Yeah I was just following the pattern used by atom 🤔 I don't know what is necessary